### ISSUE TITLE
[Add organization admin] Incorrect assistive text appears when combo box has an error

### ISSUE THREAD

**Issue by BobanL** [CDCgov/prime-simplereport#6401]
## Area or Component

Support Admin > Add Organization Admin > Organization combo box 

## Quick Summary

1. Organization combo box should have `aria-required=true`

## Accessibility Violation

https://docs.deque.com/issue-help/1.0.0/en/aria-required-missing

## Action Requested

1. Add `required=true` to the `inputProps` in [the combo box](https://github.com/trussworks/react-uswds/blob/4550708a95f955e1585d05aaf44b3592049d49b8/src/components/forms/ComboBox/ComboBox.tsx#L53)
2. Update the [assistive hint](https://github.com/trussworks/react-uswds/blob/4550708a95f955e1585d05aaf44b3592049d49b8/src/components/forms/ComboBox/ComboBox.tsx#L497-L505) when there is an error, otherwise keep the existing hint.

## Screenshots (optional)

![image](https://github.com/CDCgov/prime-simplereport/assets/10108172/994f4257-d982-4208-b032-8286ed71cdc0)

## Source Code (optional)

[which file or line is the issue attached to?]: #
